.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     timeout-minutes: 20
     strategy:
       matrix:
-        node: [8, 10, 12]
+        node: [10, 12, 14]
```

Comment:
We probably should keep Node 8 because `package.json` specifies Node >= 8.0.0. Increasing that is technically a breaking change I think.